packages/http-server/src/getHttpConfigFromRequest.ts

```diff
@@ -40,7 +40,7 @@ export const getHttpConfigFromRequest: PrismConfigFactory<IHttpConfig, IHttpRequ
     if (typeof config.mock === 'boolean') {
       config.mock = httpOperationConfig;
     } else {
-      config.mock = Object.assign({}, config.mock, httpOperationConfig);
+      config.mock = Object.assign(config.mock, httpOperationConfig);
```

Comment:
afaik `Object.assign(config.mock, httpOperationConfig);` will mutate the object ;) If you don't care then that's OK.

Reply:
Indeed I care a lot, that's why I changed.

The current line of code is mutating object anyway — so better make it explicit so I do not get a false sense of security.